content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,321 @@
+~~~~~~~~~~~~~~
+
+To generate and confirm an electronic document that will be validated by |DGI|, the sales journal
+needs to be configured with the following:
+
+- :guilabel:`Invoicing Type`: manual invoices are for open invoices previously stamped in another
+  system, for example directly in the |DGI|. Electronic invoices will not be sent to Uruware.
+- :guilabel:`Use Documents?`: Activate this option if this journal will be used to generate
+  electronic documents and sent to the government through Uruware.
+
+.. image:: uruguay/sales-journal.png
+   :align: center
+   :alt: Sales Journal for Uruguay.
+
+Workflows
+=========
+
+Once you have configured your database, you can create your documents.
+
+Sales documents
+---------------
+
+Customer invoices
+~~~~~~~~~~~~~~~~~
+
+:guilabel:`Customer invoices` are electronic documents that, when validated, are sent to |DGI| via
+Uruware. These documents can be created from your sales order or manually. They must contain the
+following data:
+
+- :guilabel:`Customer`: type the customer's information.
+- :guilabel:`Due date`: to compute if the invoice is due now or later (contado or crédito
+  respectively).
+- :guilabel:`Journal`: select the sales electronic journal.
+- :guilabel:`Document Type`: type document type in this format, for example, `(111) e-Invoice`.
+- :guilabel:`Products`: specify the product with the correct taxes.
+
+Customer credit note
+~~~~~~~~~~~~~~~~~~~~
+
+The :doc:`Customer credit note <../accounting/customer_invoices/credit_notes>` is an electronic
+document that, when validated, is sent to |DGI| via Uruware. It is necessary to have a validated
+(posted) invoice to register a credit note. On the invoice there is a button named :guilabel:`Credit
+note`, click on this button to be directed to the :guilabel:`Create credit note` form, then complete
+the following information:
+
+- :guilabel:`Reason`: type the reason for the credit note.
+- :guilabel:`Journal`: select the journal that has to be electronic and has the :guilabel:`Use
+  Documents?` option active.
+- :guilabel:`Document Type`: select the credit note document type.
+- :guilabel:`Reversal Date`: type the date.
+
+Customer debit note
+~~~~~~~~~~~~~~~~~~~
+
+The :doc:`Customer debit note <../accounting/customer_invoices/credit_notes>` is an electronic
+document that, when validated, is sent to |DGI| via Uruware. It is necessary to have a validated
+(posted) invoice to register a debit note. On the action within an invoice there is a option named
+:guilabel:`Debit note`, click on this action to be directed to the :guilabel:`Create credit note`
+form, then complete the following information:
+
+- :guilabel:`Reason`: type the reason for the credit note.
+- :guilabel:`Journal`: select the journal that has to be electronic and has the :guilabel:`Use
+  Documents?` option active.
+- :guilabel:`Copy lines`: mark checkbox to copy the invoice lines to the debit note.
+- :guilabel:`Debit note date`: type the date.
+
```

Comment:
Maybe add [this note](https://github.com/odoo/documentation/pull/9849/files#r1653641712) here, since it refers to both credit and debit notes.